docs/docs/unreachable.md

```diff
@@ -0,0 +1,90 @@
+
+### Strategy {0,0}, Lose 1 node for 4 mins
+
+Marathon is never aware and Mesos reports task.
+
+### Strategy {0,0}, Lose 1 node for 5 mins; Node becomes reachable after 6 mins of original event.
+
+At 5 min mark, Marathon will replace 1 task when TASK_UNREACHABLE status received.
+At 6 min mark, Marathon will kill the task that became reachable.
+
+### Strategy {0,300}, Lose 1 node for 5 mins; Node becomes reachable after 6 mins of original event.
+
+At 5 min mark, Marathon will replace 1 task when TASK_UNREACHABLE status received.
+At 6 min mark, Marathon reports number of tasks as 2 of 1 for application.
+Somewhere between the 10 min mark and 20 min mark, the reachable task will be killed and Marathon reports 1 of 1 for application.
+
+**Note:** The expunge time is 300 seconds which is 5 mins. 5 mins after Marathon was notified of the unreachable event is the 10 min mark.
+It is possible that the reconciliation internal just happened prior to the 10 min mark and the next time to expunge is the next reconciliation time which is at the 20 min mark.
+
+### Strategy {0,0}, Lose 4 nodes for hours, the task node is last.
+
+At 5 min mark, node 1 would have an unreachable event.
+At 25 min mark, node 2 would have an unreachable event.
+At 45 min mark, node 3 would have an unreachable event.
+At 65 min mark, Marathon will replace the task when TASK_UNREACHABLE status received.
+
+### Strategy {300,300}, Lose 4 nodes for hours, the task node is last.
+
+At 5 min mark, node 1 would have an unreachable event.
+At 25 min mark, node 2 would have an unreachable event.
+At 45 min mark, node 3 would have an unreachable event.
+At 65 min mark, Marathon receive TASK_UNREACHABLE status for task.
+Somewhere between 70 mins and 80 mins, Marathon will replace the unreachable task.
+
+### Strategy {0,0}, Lose 1 node(no task) for 5 mins and returns at 6 min mark. Lose task node for for undefined amount of time.
+
+At 5 min mark, node 1 would have an unreachable event.
+At 25 min mark, Marathon will replace the task when TASK_UNREACHABLE status received.
+
+### Strategy {86400,86400}, Lose 1 node for 25 hours
+
+At 5 min mark, Marathon is aware that task is unreachable.
+Somewhere between 24h5m and 24h15m, Marathon will replace the unreachable task.
+
+## Unreachable Summary
+
+There are 3 Windows of time when dealing with unreachable tasks. The first window is the time it takes Marathon to be notified of a task being unreachable. This is dependent on the number of health check failing nodes leading up to the event with a 5 min minimum. When Marathon is notified it will respond within another window of time. A task replacement window starts with the inactiveAfterSeconds time which could be immediately and up to the reconciliation time window. The final window is expunging an unreachable task that has become reachable again. This window could be immediately as the task is reachable if the expunge time has elapsed.
```

Comment:
> This window could be immediately as the task is reachable if the expunge time has elapsed.
`s/immediately/immediate`

Again: I don't it's true - if an unreachable task becomes reachable again we immediately decide on which one of both (new or old) should be killed (`expungeAfterSeconds` doesn't play a role here). See this test:
mesosphere/marathon/integration/TaskUnreachableIntegrationTest.scala:41

Reply:
`Windows -> windows`